sdk/loadtestservice/azure-developer-loadtesting/azure/developer/loadtesting/aio/operations/_operations.py

```diff
@@ -45,12 +45,12 @@
     build_test_run_create_or_update_server_metrics_config_request,
     build_test_run_delete_test_run_request,
     build_test_run_get_app_components_request,
+    build_test_run_get_metric_definitions_request,
```

Comment:
can you add a changelog for these changes?

Reply:
added with the latest commit